source/whats-new.txt

```diff
@@ -73,6 +74,22 @@ New versions of the {+driver-short+} can introduce the following types of breaki
      - Drops support for platform A, compiler B, or architecture C.
 
 .. _upcoming-breaking-changes:
+.. _version-4.0:
+
+What's New in 4.0
+-----------------
+
+.. warning:: Breaking Changes
```

Comment:
```suggestion
.. warning:: Build System Breaking Changes
```
We distinguish and group build system breaking changes separately from API and ABI breaking changes.

Note: it seems like the 3.11 section also needs to be split accordingly (paraphrased for brevity):

---

> WARNING: Build System Breaking Changes
> The v3.11 driver introduces the following breaking changes to the build system:
> - drop server 3.6
> - require C Driver 1.28.0+
>
> WARNING: ABI Breaking Changes
> The v3.11 driver introduces the following breaking changes to the ABI:
> - removed exports
> - declare with `__cdecl`.